### DISCUSSION TITLE
Add a way to filter out really old articles

### DISCUSSION THREAD

**Discussion by jaoord** [dailydotdev/daily#1567] [Feature Requests]
Please allow us to filter out articles that are older than X weeks or months. I would like to see a fresh feed. If there are other ways to achieve this, please let me know.

**Comment by idoshamun** [comment]
Hi @jaoord,

Have you considered https://app.daily.dev/posts/latest?

**Reply by jaoord** [reply]
I hadn't seen the big buttons on the top. Thanks! that will do
